src/pubsub/ua_pubsub_datasetmetadata_json.c

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2022 ISW (for umati and VDW e.V.) (Author: Moritz Walker)
+ */
```

Comment:
The function isn't that long and i think it will not grow that much in the future. I would preffer to add it to the networkmessage layer.

Reply:
Done